src/test/resources/org/jabref/logic/crawler/study.yml

```diff
@@ -0,0 +1,16 @@
+authors:
+  - Jab Ref
+studyName: TestStudyName
+lastSearchDate: 2020-11-26
+researchQuestions:
+  - Question1
+  - Question2
+queries:
+  - query: Quantum
+  - query: Cloud Computing
+  - query: '"Software Engineering"'
+libraries:
```

Comment:
Maybe `databases` or `sources` is slightly better than `libraries` ? Arxiv or springer for example are not really libraries.